### DISCUSSION TITLE
401 Unauthorized

### DISCUSSION THREAD

**Discussion by Magonox** [danielmiessler/Fabric#553] [Q&A]
I everyone, 
after completing the installation of fabric  i am getting the following:

pbpaste | fabric --pattern summarize
Error: Client error '401 Unauthorized' for url 'https://api.openai.com/v1/models'
For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401

I have updated my 

~/.config/fabric/.env 

with the api key, however I am still running into the same problem.

**Comment by mmartoccia** [comment]
+1 as I am having this problem on a different machine and a fresh install. My original, local install, works fine and no issues. Both MacOS same installation procedures, same .env and keys.

**Reply by Magonox** [reply]
I agree. I installed it on my Mac Pro without a problem, but it's just not working on my Mac Mini (I don't know how many times I reinstalled .. but will try again)
